Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/ConsoleLoggerWriter.cs

```diff
@@ -202,25 +217,32 @@ internal void FormattedWriteLine(string message)
 
             internal void FormattedWriteLine(string level, string message)
             {
-                string line;
-                if(!string.IsNullOrEmpty(level))
+                var displayLevel = level;
+                if (Enum.TryParse<LogLevel>(level, true, out var levelEnum))
                 {
-                    var displayLevel = level;
-                    if (Enum.TryParse<LogLevel>(level, true, out var levelEnum))
-                    {
-                        if (levelEnum < _minmumLogLevel)
-                            return;
+                    if (levelEnum < _minmumLogLevel)
+                        return;
 
-                        displayLevel = ConvertLogLevelToLabel(levelEnum);
-                    }
+                    displayLevel = ConvertLogLevelToLabel(levelEnum);
+                }
 
-                    line = $"{DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.fffZ")}\t{CurrentAwsRequestId}\t{displayLevel}\t{message ?? string.Empty}";
+                if (_logFormatType == LogFormatType.Unformatted)
+                {
+                    _innerWriter.WriteLine(message);
                 }
                 else
                 {
-                    line = $"{DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss.fffZ")}\t{CurrentAwsRequestId}\t{message ?? string.Empty}";
+                    string line;
+                    if (!string.IsNullOrEmpty(level))
```

Comment:
Should this be `displayLevel` instead of `level`?

Reply:
Switched to `displayLevel` although the results will be the same.